### DISCUSSION TITLE
Quarkus 3.2.5.Final released - Maintenance release

### DISCUSSION THREAD

**Discussion by giscus** [quarkusio/quarkus#36498] [Quarkus Blog/Website]
# Quarkus 3.2.5.Final released - Maintenance release

Quarkus: Supersonic Subatomic Java

https://quarkus.io/blog/quarkus-3-2-5-final-released/

**Comment by timonzi** [comment]
Is there any plan when we will get a new release for the LTS version? So 3.2.7.Final? We are still waiting for some bug fixes that were reported and fixed more than a month ago.

**Reply by gsmet** [reply]
@timonzi which bugfixes specifically? We're gonna build a new 3.2.7 soon but we will be a bit conservative again because we primarily want to fix the ongoing CVEs without taking too many risks. But let me know which issues are blocking for you and I'll see if they are low risk enough to get included in 3.2.7.
We have plans for a 3.2.8 with more fixes in once the important CVEs are behind us.

**Reply by timonzi** [reply]
@gsmet For us the most important issue would be: [35774](https://github.com/quarkusio/quarkus/issues/35774)

An update of Hibernate would also be nice in order to get fix for e.g. [HHH-16979](https://hibernate.atlassian.net/browse/HHH-16979). However, since we have already refactored our code or added workarounds for Hibernate issues that have occurred since version 6, the relevance is currently not that high. Also, several issues in Hibernate are not yet resolved - i.e. we still have to wait either way until we can remove the workarounds.

**Reply by gsmet** [reply]
I will try to include the fix for https://github.com/quarkusio/quarkus/issues/35774 .

As for Hibernate ORM, it's a bit more complex so this will have to wait for 3.2.8.

**Reply by timonzi** [reply]
Thank you :+1:
